pom.xml

```diff
@@ -6,7 +6,7 @@
 	<groupId>SynapseAWSConsoleLogin</groupId>
 	<artifactId>SynapseAWSConsoleLogin</artifactId>
 	<packaging>war</packaging>
-	<version>1.0</version>
+	<version>develop-SNAPSHOT</version>
```

Comment:
i assume this will generate a `synapse-login-scipool-develop-SNAPSHOT.jar`?  the resulting artifact must match the what is referenced in the beanstalk deployment file  https://github.com/Sage-Bionetworks/synapse-login-scipool/blob/develop/.elasticbeanstalk/config.yml#L10

Reply:
This is a placeholder that will get replaced by either the last git tag (prod) or the last commit (dev) when pomSnapshot*.sh is run. I'll check on the effect on the artifact name.